Model/Quote/Checkout.php

```diff
@@ -97,6 +97,20 @@ public function getCheckoutSettings()
         ];
     }
 
+    /**
+     * @return float|int
+     */
+    private function getDeliveryDelayDay()
+    {
+        $productDelay = $this->package->dropOffDelayDayWithProduct($this->products);
+        $generalDelay = $this->helper->getIntergerConfig('general/dropoff_delay');
+
+        if (! $productDelay) {
+            $productDelay = $generalDelay;
```

Comment:
```suggestion
            $productDelay = $this->helper->getIntergerConfig('general/dropoff_delay');
```
En dan kan regel 106 weghalen

Reply:
done